storagetest/storagetest.go

```diff
@@ -2,69 +2,232 @@ package storagetest
 
 import (
```

Comment:
Diff isn't beautiful. It was almost 100% replaced. Look here: https://github.com/giantswarm/microstorage/blob/d7a8587438a752f2f7db45a3d9d367a0101171e4/storagetest/storagetest.go